backend/windmill-api/openapi.yaml

```diff
@@ -1608,7 +1608,7 @@ paths:
                 type: object
```

Comment:
All the clients rely on the location of this file so I believe some changes in their generator would be needed if we were to move this.

Reply:
Which do you prefer, moving it back or adjusting references?

Reply:
Let's adjust the references, I think it makes more sense in windmill-api